src/Uno.UI/UI/Xaml/Controls/CalendarView/CalendarView_Partial.cs

```diff
@@ -1848,7 +1849,12 @@ internal void CoerceDate(ref DateTime date)
 
 				if (isScopeChanged)
 				{
+#if __ANDROID__
+					// .InvalidateMeasure() bug
```

Comment:
Can you open an issue and reference it here ?

Reply:
I already did that, but for some reason this change did't reach the last commit.

Reply:
Done...